components/Pagination.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { BlogPostResponse } from '../apis/contentful';
+import Link from 'next/link';
+
+interface PaginationProps extends Omit<BlogPostResponse, 'limit'> {}
+
+const Pagination = ({ entries, total, skip }: PaginationProps) => {
```

Comment:
Does `skip` mean the current page? If so, I think we should name it `current` instead. Also, I think it'd be better for a pagination to contain 3 props:
- `current`: current page number
- `pageSize`: number of items per page
- `total`: total number of items

Then doing the render based on the three numbers.